tests/CMakeLists.txt

```diff
@@ -432,7 +434,7 @@ function(newtest_gcc test source depender gcc_ops test_ops drmem_ops dr_ops
 endfunction(newtest_gcc)
 
 # i#1726: ARM is only supported in pattern mode.
-if (NOT ARM)
+if (NOT ARM AND NOT AARCH64)
```

Comment:
Please update the comment which only talks about ARM: probably it should reference i#2016 for AArch64 right?

Reply:
Looks done.